app/views/landing_page/blocks/_main_navigation.html.erb

```diff
@@ -27,10 +27,12 @@
                 <% link[:items].each do |child_link| %>
                   <%
                     child_link_classes = "app-b-main-nav__link govuk-link govuk-link--no-underline govuk-link--no-visited-state"
-                    child_link_classes << " app-b-main-nav__link--active" if child_link[:active]
+                    child_link_aria = { current: true } if child_link[:active]
+                    child_li_classes = "app-b-main-nav__listitem"
+                    child_li_classes << " app-b-main-nav__listitem--active" if child_link[:active]
                   %>
-                  <li class="app-b-main-nav__listitem">
-                    <%= tag.a(href: child_link[:href], class: child_link_classes) do %>
+                  <li class="<%= child_li_classes %>">
```

Comment:
Could we use `<%= tag.li(class: child_li_classes) do %>` here for consistency with the other elements in this template that interface with ruby?

Reply:
Sure, will make that change 👍